experimental/traffic-portal/src/app/api/testing/delivery-service.service.ts

```diff
@@ -536,4 +544,20 @@ export class DeliveryServiceService {
 	public async getDSTypes(): Promise<Array<TypeFromResponse>> {
 		return this.dsTypes;
 	}
+
+	/**
+	 * Gets a Delivery Service's SSL Keys
+	 *
+	 * @param ds The delivery service xmlid or object
+	 * @returns The DS ssl keys
+	 */
+	public async getSSLKeys(ds: string | ResponseDeliveryService): Promise<ResponseDeliveryServiceSSLKey> {
+		const xmlId = typeof ds === "string" ? ds : ds.xmlId;
+		const key = this.dsSSLKeys.find(k => k.deliveryservice === xmlId);
+		if(!key) {
+			throw new Error(`no such Delivery Service: ${ds}`);
```

Comment:
the string-ification of an arbitrary object is just `[object Object]` which doesn't tell you anything about which DS doesn't exist - this should use the XMLID instead